### PR TITLE
Cleanse functionality from relying on `yfinance`

### PR DESCRIPTION
**Pull request by JerBouma**
# Description

This tackles most of the items listed in #4169 and more. Only makes adjustments where the functionality simply was broken and it made sense to replace. This adjusts the following:

- Remove `po/dividend` and `po/property` since full reliance on Yahoo Finance as well as not having any merit to it. Adjust `po/mktcap` and `po/blacklitterman` to rely on FinancialModelingPrep for obtaining market cap instead of Yahoo Finance. The`po/equal` command can stay the same.
- Adjust `etf/news` so it just searches the ticker. This actually leads to decent results.
- Adjust `etf/weights` so it uses FinancialModelingPrep instead.
- Fixed issues with `portfolio` menu not having data and `portfolio/alloc` works again, fixing #4179. All reliance on `yfinance` (except for market data) is now gone.
- Delete:
    - `stocks/fa/sust` command since it doesn't work and we couldn't find a free alternative. Fixing #4168
    - `stocks/quote --source YahooFinance` command since it doesn't work and we have an alternative.
    - `etf/summary` command directly relied on Yahoo Finance.
    - `etf/pir` command. Since it's my own package, sad to see it go but it just has too much reliance on Yahoo Finance that it is going to always break pretty quick.

Misc:

- Export now works on `stocks/quote`.
- `etf/load` command no longer displays holdings. If you request too much of StockAnalysis they will block you for a while therefore I've taken this out, it doesn't add much value to have it if there is a command that does the same but with much more detail.
- Added sources to `portfolio/po` so people are aware if they run `mktcap` it will semi-drain their free FMP key.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [x] Make sure affected commands still run in terminal
- [x] Ensure the SDK still works
- [x] Check any related reports

# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by JerBouma** [comment]
Ok @jmaslek this should be about done now. This is also my time to shift back to Academia focus and stop being one of your Engineers 😛

**Comment by jmaslek** [comment]
> Ok @jmaslek this should be about done now. This is also my time to shift back to Academia focus and stop being one of your Engineers 😛

we will miss you

File: openbb_terminal/etf/fmp_model.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    name: str
+        ETF name
+
+    Returns
+    -------
+    Dict[str, Any]
+        Dictionary with sector weightings allocation
+    """
+    try:
+        response = urlopen(
```
Comment (jmaslek) [inline openbb_terminal/etf/fmp_model.py:34]: really had to be difficult and go with urllib instead of requests lol

Comment (JerBouma) [reply openbb_terminal/etf/fmp_model.py:34]: Oh lol actually just copy pasted from FundamentalAnalysis package

File: openbb_terminal/common/newsapi_view.py [hunk 2]
```diff
@@ -46,6 +47,7 @@ def display_news(
     if tables:
         for table in tables:
             print_rich_table(table[0], title=table[1]["title"])
+            console.print("\n")
```
Comment (jmaslek) [inline openbb_terminal/common/newsapi_view.py:50]: too many newlines.  console.print() already newlines

File: openbb_terminal/etf/fmp_model.py (hunk 1 above)
Comment (jmaslek) [reply openbb_terminal/etf/fmp_model.py:34]: is this a free endpoint or nah ?

Comment (JerBouma) [reply openbb_terminal/etf/fmp_model.py:34]: Yeah it's free but for some tickers it aint.

Comment (JerBouma) [reply openbb_terminal/etf/fmp_model.py:34]:
```
(🦋) /etf/ $ load VOO

Loading Daily data for VOO with starting period 2022-02-08. 


(🦋) /etf/ $ weights


(🦋) /etf/ $ weights --raw

                                   
      Sector holdings of VOO       
┏━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃ Sector                 ┃ Weight ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ Real Estate            │ 2.47%  │
├────────────────────────┼────────┤
│ Consumer Cyclical      │ 12.21% │
├────────────────────────┼────────┤
│ Basic Materials        │ 2.34%  │
├────────────────────────┼────────┤
│ Consumer Defensive     │ 6.59%  │
├────────────────────────┼────────┤
│ Technology             │ 23.5%  │
├────────────────────────┼────────┤
│ Communication Services │ 10.93% │
├────────────────────────┼────────┤
│ Financial Services     │ 14.18% │
├────────────────────────┼────────┤
│ Utilities              │ 2.65%  │
├────────────────────────┼────────┤
│ Industrials            │ 9.22%  │
├────────────────────────┼────────┤
│ Energy                 │ 2.79%  │
├────────────────────────┼────────┤
│ Healthcare             │ 13.13% │
└────────────────────────┴────────┘

(🦋) /etf/ $ load SPY

Loading Daily data for SPY with starting period 2022-02-08. 


(🦋) /etf/ $ weights --raw

                                   
      Sector holdings of SPY       
┏━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃ Sector                 ┃ Weight ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ Real Estate            │ 2.53%  │
├────────────────────────┼────────┤
│ Consumer Cyclical      │ 12.43% │
├────────────────────────┼────────┤
│ Basic Materials        │ 2.34%  │
├────────────────────────┼────────┤
│ Consumer Defensive     │ 6.4%   │
├────────────────────────┼────────┤
│ Technology             │ 23.44% │
├────────────────────────┼────────┤
│ Communication Services │ 11.18% │
├────────────────────────┼────────┤
│ Financial Services     │ 14.41% │
├────────────────────────┼────────┤
│ Utilities              │ 2.64%  │
├────────────────────────┼────────┤
│ Industrials            │ 9.05%  │
├────────────────────────┼────────┤
│ Energy                 │ 2.68%  │
├────────────────────────┼────────┤
│ Healthcare             │ 12.89% │
└────────────────────────┴────────┘

(🦋) /etf/ $ load VTI

wLoading Daily data for VTI with starting period 2022-02-08. 


(🦋) /etf/ $ weights --raw

                                   
      Sector holdings of VTI       
┏━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃ Sector                 ┃ Weight ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ Real Estate            │ 3.51%  │
├────────────────────────┼────────┤
│ Consumer Cyclical      │ 12.08% │
├────────────────────────┼────────┤
│ Basic Materials        │ 2.54%  │
├────────────────────────┼────────┤
│ Consumer Defensive     │ 6.05%  │
├────────────────────────┼────────┤
│ Technology             │ 22.9%  │
├────────────────────────┼────────┤
│ Communication Services │ 10.29% │
├────────────────────────┼────────┤
│ Financial Services     │ 13.98% │
├────────────────────────┼────────┤
│ Utilities              │ 2.51%  │
├────────────────────────┼────────┤
│ Industrials            │ 9.78%  │
├────────────────────────┼────────┤
│ Energy                 │ 2.64%  │
├────────────────────────┼────────┤
│ Healthcare             │ 13.73% │
└────────────────────────┴────────┘

(🦋) /etf/ $
```

Comment (jmaslek) [reply openbb_terminal/etf/fmp_model.py:34]: nice.  was curious

**Comment by jmaslek** [comment]
> Adjust etf/news so it just searches the ticker. This actually leads to decent results.

lol @ searching for spy

**Comment by JerBouma** [comment]
> > Adjust etf/news so it just searches the ticker. This actually leads to decent results.
> 
> lol @ searching for spy

Haha lol, I just used things like `VTI`, `VOO` etc. Those work nicely. Whatever, nice easter egg 😛

**Comment by jmaslek** [comment]
sort plz
```
      Sector holdings of SPY       
┏━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
┃ Sector                 ┃ Weight ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
│ Real Estate            │ 2.53%  │
├────────────────────────┼────────┤
│ Consumer Cyclical      │ 12.43% │
├────────────────────────┼────────┤
│ Basic Materials        │ 2.34%  │
├────────────────────────┼────────┤
│ Consumer Defensive     │ 6.4%   │
├────────────────────────┼────────┤
│ Technology             │ 23.44% │
├────────────────────────┼────────┤
│ Communication Services │ 11.18% │
├────────────────────────┼────────┤
│ Financial Services     │ 14.41% │
├────────────────────────┼────────┤
│ Utilities              │ 2.64%  │
├────────────────────────┼────────┤
│ Industrials            │ 9.05%  │
├────────────────────────┼────────┤
│ Energy                 │ 2.68%  │
├────────────────────────┼────────┤
│ Healthcare             │ 12.89% │
└────────────────────────┴────────┘
```

**Comment by jmaslek** [comment]
> etf/pir command. Since it's my own package, sad to see it go but it just has too much reliance on Yahoo Finance that it is going to always break pretty quick.

did you remove the dep?

**Comment by JerBouma** [comment]
> > etf/pir command. Since it's my own package, sad to see it go but it just has too much reliance on Yahoo Finance that it is going to always break pretty quick.
> 
> did you remove the dep? hold off on it for now while the deps are being figured out.

Fixed the sort and didn't remove the dep.

**Pull request merged**